ui/app/serializers/clients/activity.js

```diff
@@ -45,7 +86,6 @@ export default class ActivitySerializer extends ApplicationSerializer {
         non_entity_clients,
       };
     }
-    return object;
```

Comment:
What happens if the object doesn't include any of the keys we check for? Return an empty object?